backend/app/api/endpoints/knowledge.py

```
@@ -914,6 +923,130 @@ def batch_delete_documents(
+
+
+@router.put(
+    "/{knowledge_base_id}/folders/{folder_id}",
+    response_model=KnowledgeFolderResponse,
+)
+@trace_sync("update_folder", "knowledge.api")
+def update_folder(
+    knowledge_base_id: int,
+    folder_id: int,
+    data: KnowledgeFolderUpdate,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)